src/.vuepress/components/ExtensionList.vue

```diff
@@ -1,16 +1,50 @@
 <template>
 	<div class="extension-list">
-		<div v-for="extensionGroup in extensions" :key="extensionGroup[0].lang">
+		<span class="filters-list">
+			<ElInput v-model="filters.search" placeholder="Search extensions by name..." clearable />
+
+			<ElSelect v-model="filters.lang" placeholder="Show specific languages..." multiple clearable>
+				<ElOption
+					v-for="[group] in extensions"
+					:key="group.lang"
+					:label="group.lang === 'en' ? simpleLangName(group.lang) : langName(group.lang)"
+					:value="group.lang"
+				/>
+			</ElSelect>
+
+			<div>
+				Sort by
+				<ElRadioGroup v-model="filters.sort">
+					<ElRadioButton label="ASC"></ElRadioButton>
+					<ElRadioButton label="DESC"></ElRadioButton>
```

Comment:
Probably better to go with `Ascending` and `Descending` because everyone is not native English speakers or they have poor English skills
```suggestion
					<ElRadioButton label="Ascending"></ElRadioButton>
					<ElRadioButton label="Descending"></ElRadioButton>
```

Reply:
https://github.com/tachiyomiorg/website/pull/636/commits/d6ed8a6ce05e1d5f5a53d9f55688ad12ca5cc1f9